packages/emnapi/include/emnapi.h

```diff
@@ -15,6 +15,22 @@ typedef enum {
   emnapi_userland,
 } emnapi_ownership;
 
+typedef enum {
+  emnapi_int8_array,
+  emnapi_uint8_array,
+  emnapi_uint8_clamped_array,
+  emnapi_int16_array,
+  emnapi_uint16_array,
+  emnapi_int32_array,
+  emnapi_uint32_array,
+  emnapi_float32_array,
+  emnapi_float64_array,
+  emnapi_bigint64_array,
+  emnapi_biguint64_array,
+  emnapi_data_view = 16,
```

Comment:
Why explicit numbers here?

Reply:
Considering that `napi_typedarray_type` may increase member in the future, want to keep the same value